src/i_main.c

```diff
@@ -112,6 +113,8 @@ int main(int argc, char **argv)
    myargc = argc;
    myargv = argv;
 
+   setlocale(LC_ALL, ".UTF8");
```

Comment:
Should we check for a return value? What do we expect if changing to UTF-8 does not work?

Reply:
> What do we expect if changing to UTF-8 does not work?

C functions will work in ASCII mode witch is fine for us. UCRT can be run starting from Windows XP.